src/Core/Prefab/GlobeView.js

```diff
@@ -399,15 +418,14 @@ GlobeView.prototype.getPickingPositionFromDepth = function getPickingPositionFro
     const prev = this.camera.camera3D.layers.mask;
     this.camera.camera3D.layers.mask = 1 << this.wgs84TileLayer.threejsLayer;
 
-    const previousRenderState = this._renderState;
-    this.changeRenderState(RendererConstant.DEPTH);
-
-    // Render to buffer
-    var buffer = this.mainLoop.gfxEngine.renderViewTobuffer(
-        this,
-        this.mainLoop.gfxEngine.fullSizeRenderTarget,
-        mouse.x, dim.y - mouse.y,
-        1, 1);
+    let buffer;
+    if (viewPaused) {
+        fullSizeDepthBuffer = fullSizeDepthBuffer || this.getDepthBuffer(0, 0, dim.x, dim.y);
+        const id = ((dim.y - mouse.y) * dim.x + mouse.x) * 4;
```

Comment:
Are you sure about this?
If `mouse.y` is `0`, you're going to read `dim.y * dim.x + mouse.x` which is probably past the end of the buffer.

Reply:
yes it's normal , the y is reversed in buffer
see in master, it's reversed : https://github.com/iTowns/itowns/pull/474/files/193b6928ce34b331577af08120514e066359ff5a#diff-caf5227be4f9e2f283d82df32a678e4bL408

Reply:
I'm not asking about the reversed y axis.

Your buffer size is `dim.x * dim.y * 4`
If mouse is: `{ x: 10, y: 0 }` you're going to start reading `(dim.y * dim.x + 10) * 4` which is out of bounds (ie: I guess the correct formula is `((dim.y - mouse.y - 1) * dim.x + mouse.x) * 4`)

Reply:
yes, thanks